### ISSUE TITLE
Drop support for some older versions of PHP and Laravel in the next major release

### ISSUE THREAD

**Issue by adiachenko** [Vinelab/tracing-laravel#41]
At the moment our versions constraints are brittle and its getting harder to anticipate where things might break due to wide range of supported software.

I suggest to limit support to the following versions to use maintenance burden:

- PHP >= 7.4 (support for PHP 7.3 will end on December 6)
- Laravel >= 6.0 (this is the last version that still receives security fixes)

This also allows us to clean up some other dependencies a bit:
- "openzipkin/zipkin": `"~2.0.2|~3.0"` => `"~3.0"`
- "phpunit/phpunit": `"~7.0|~8.0"` => `"~9.0"`

Users of older PHP/Laravel can still avail of older versions of the package.

Opinions? @Mulkave @KinaneD

**Comment by jcchavezs** [comment]
+1

**Comment by KinaneD** [comment]
+1

**Comment by KinaneD** [comment]
Though I just noticed that Zipkin 3.0 has not been released yet.

**Comment by jcchavezs** [comment]
It is been released today.

**Comment by adiachenko** [comment]
Yeah, Zipkin 3 has been released and there is #40 on review to tag our own v2.1.0 (final) release that includes it.

**Comment by KinaneD** [comment]
It was indeed.
Thank you @adiachenko @jcchavezs 🦾

**Comment by Mulkave** [comment]
Good idea 👍
